core/domain/email_manager.py

```diff
@@ -77,6 +77,34 @@ def log_new_error(*args, **kwargs):
         'subject': _PLACEHOLDER_SUBJECT,
         'html_body': _PLACEHOLDER_HTML_BODY,
     })
+
+EXPLORATION_ROLE_MANAGER = 'Manager rights'
```

Comment:
These should not be capitalized? (It'll become more obvious once you write the "golden" email.)